messaging/propagate.go

```diff
@@ -161,19 +163,29 @@ func (p *Propagate) Dispatch() error {
 			}
 			if !p.IsRoot() {
 				log.Lvl3(p.ServerIdentity(), "Sending to parent")
-				p.SendToParent(&PropagateReply{})
+				if err := p.SendToParent(&PropagateReply{}); err != nil {
+					return err
+				}
 			}
 			if p.IsLeaf() {
 				process = false
 			} else {
 				log.Lvl3(p.ServerIdentity(), "Sending to children")
-				p.SendToChildrenInParallel(&msg.PropagateSendData)
+				if err := p.SendToChildrenInParallel(&msg.PropagateSendData); err != nil {
```

Comment:
It's ok have some failures here as long as it's not more than `allowedFailures`

Reply:
It's not in R4M yet ;) But thanks for the heads-up.